### PR TITLE
fix: fetch body

### PR DESCRIPTION
**Pull request by PeytonHanel**
fix: Provided fetch function accepts the same object types in the body as the native fetch function

See #9420 

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

**Comment by PeytonHanel** [comment]
I'm not sure why it says no changeset found, I generated one and it's in the `files changed` section.

**Comment by Conduitry** [comment]
The changeset needs to be created from the root of the repo. It looks like you initialized another changesets installation within the `packages/kit` directory.

File: .changeset/thick-gorillas-judge.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Fix: Sveltekit's provided fetch function now accepts the same input types for the body as the native fetch function.
```
Comment (s3812497) [inline .changeset/thick-gorillas-judge.md:5]:
```suggestion
fix: Sveltekit's provided fetch function now accepts the same input types for the body as the native fetch function.
```

**Comment by Rich-Harris** [comment]
For posterity: it looks like the reason this is okay is that we just don't serialize `fetch` responses in the case where `request.body` exists and isn't a string:

https://github.com/sveltejs/kit/blob/4e106400a20dd3a496cb9aaff0eaa0f5324c452a/packages/kit/src/runtime/server/page/load_data.js#L234

**Review by Rich-Harris — APPROVED** [review]

**Comment by Rich-Harris** [comment]
thank you!

**Pull request merged**